### PR TITLE
Enable database and create migration script for the schema

### PR DESCRIPTION
**Pull request by AmritSidhu**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 version: '3.2'
 services:
-#  db:
-#    image: postgres
-#    volumes:
-#      - ./tmp/db:/var/lib/postgresql/data
+  db:
+    image: postgres
```
Comment (galund) [inline docker-compose.yml]: Can you have a look and see if there's a way to specify the postgres version? Would be good to lock that in to the same as the PaaS version ie 10.

Comment (AmritSidhu) [reply docker-compose.yml]: postgres:10 is the name of the image and the version separated by a colon which ive now implemented

Comment (AmritSidhu) [reply docker-compose.yml]: So turns out we want postgres:11, but we can set it to psotgres:latest?

Comment (galund) [reply docker-compose.yml]: Oh, I'm confused: we chose `tiny-unencrypted-10` for the PaaS service, so I think 10 is fine?

Comment (AmritSidhu) [reply docker-compose.yml]: When trying to run it, it says it needs the latest version of postgres which is 11

Comment (galund) [reply docker-compose.yml]: In which case I wonder how this works when we deploy to PaaS? We need these to be consistent, somehow, for sure.

**Pull request merged**